sources/packages/web/src/assets/css/base.scss

```diff
@@ -102,3 +108,7 @@ ul.no-bullets {
   list-style-type: none;
   margin-bottom: 0.5rem;
 }
+
+.p-tabview-panels {
```

Comment:
is this class used in the PR, I didn't find it

Reply:
This class belongs to Primevue. It is to create the space between tabview panel and tabview nav.